packages/hoprd/src/commands/sign.ts

```diff
@@ -0,0 +1,32 @@
+import type Hopr from '@hoprnet/hopr-core'
+import { u8aToHex } from '@hoprnet/hopr-utils'
+
+import { AbstractCommand } from './abstractCommand'
+import { styleValue } from './utils'
+
+export default class Sign extends AbstractCommand {
+  constructor(public node: Hopr) {
+    super()
+  }
+
+  public name() {
+    return 'sign'
+  }
+
+  public help() {
+    return 'Signs any arbitrary message with a node’s key'
```

Comment:
Same here. *Arbitrary* is certainly the wrong word. Please come up with a different description.

Reply:
Indeed, updating.